rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/nodesource/infrastructure/SSHInfrastructureV2.java

```diff
@@ -109,6 +111,12 @@
     @Configurable(description = "Options for the java command launching the node on the remote hosts", sectionSelector = 3)
     protected String javaOptions;
 
+    @Configurable(description = "Agent, Jar or NoStartup. Please refer to the manual", sectionSelector = 1)
```

Comment:
It is difficult to explain and understand here what Agent, Jar and NoStartup means.

It would be easier if we had two extra parameters startupScriptStandard and startupScriptWithNodeJarDownload

In that case deploymentMode could be: useStartupScript, useNodeJarStartupScript and Autogenerated. Which seems easier to explain